operator/cmd/mesh/install.go

```diff
@@ -100,6 +101,13 @@ func InstallCmd(logOpts *log.Options) *cobra.Command {
   istioctl install --set "values.sidecarInjectorWebhook.injectedAnnotations.container\.apparmor\.security\.beta\.kubernetes\.io/istio-proxy=runtime/default"
 `,
 		Args: cobra.ExactArgs(0),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			errs := validation.IsQualifiedName(iArgs.revision)
+			if len(errs) != 0 && cmd.PersistentFlags().Changed("revision") {
+				return fmt.Errorf("invali revision specified: %v", errs)
```

Comment:
nit: I think passing a []string to a fmt.*f function just inlines the entire slice, so this might result in one 500 character wide line.

You might want to do something like

```
return fmt.Errorf("invalid revision specified:\n%v", strings.Join(errs, "\n"))
```

but it's up to you.